.readthedocs.yml

```diff
@@ -1,8 +1,9 @@
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 version: 2
 sphinx:
-  configuration: docs/conf.py
```

Comment:
Are you sure we want to kill the KFP documentation.
Perhaps there can be a team discussion about this?

Reply:
This PR does not release to master, it merges to the release branch for google-cloud-pipeline-components.  That way we can maintain the documentation for google-cloud-pipeline-components and KFP SDK in parallel.

Reply:
I'd second Ark-kun's point here, we can try, but I worry there could be unexpected side effects for using different branches to distinguish KFP and google cloud components.

Can we create a tracking issue to find a long term solution? Is there no way to host two readthedocs pages in the same repo? I assume monorepos are quite popular now, there should be a way around than this.